### PR TITLE
Update health check task.

### PR DESCRIPTION
**Pull request by myidpt**

### CODE REVIEW

File: content/docs/tasks/security/health-check/index.md [hunk 1]
```diff
@@ -5,15 +5,11 @@ weight: 70
 keywords: [security,health-check]
 ---
 
-This task shows how to enable Kubernetes health checking for Citadel. Note this is an Alpha feature.
-
-Since Istio 0.6, Citadel has a health checking feature that can be optionally enabled.
-By default, the normal Istio deployment process does not enable this feature.
+Citadel has a health checking feature that can be optionally enabled.
```
Comment (rcaballeromx) [inline content/docs/tasks/security/health-check/index.md]:
```suggestion
You can enable Citadel's health checking feature
```

File: content/docs/tasks/security/health-check/index.md [hunk 2]
```diff
@@ -5,15 +5,11 @@ weight: 70
...
 Currently, the health checking feature is able to detect the failures of Citadel CSR signing service,
```
Comment (rcaballeromx) [inline content/docs/tasks/security/health-check/index.md]: What does CSR stand for? Please expand abbreviations when you introduce them in a document. You can use the abbreviation on the following instances.
```suggestion
to detect the failures of the Citadel CSR signing service.
```

File: content/docs/tasks/security/health-check/index.md [hunk 3]
```diff
@@ -5,15 +5,11 @@ weight: 70
...
-by periodically sending CSRs to the API. More health checking features are coming shortly.
+by periodically sending CSRs to the API and verifying the response.
```
Comment (rcaballeromx) [inline content/docs/tasks/security/health-check/index.md]: What API are you referring to? Do you mean the reader's service? 
```suggestion
Citadel periodically sends CSRs to the API and verifies the response.
```

File: content/docs/tasks/security/health-check/index.md [hunk 4]
```diff
@@ -5,15 +5,11 @@ weight: 70
...
 
-Citadel contains a _prober client_ module that periodically checks Citadel's status (currently only the health
-status of the gRPC server).
+Citadel contains a _prober client_ module that periodically checks the health status of Citadel's gRPC server.
```
Comment (rcaballeromx) [inline content/docs/tasks/security/health-check/index.md]:
```suggestion
The _prober client_ module in Citadel periodically checks the health status of Citadel's gRPC server.
```

File: content/docs/tasks/security/health-check/index.md [hunk 5]
```diff
@@ -22,74 +18,51 @@ If the file is not updated for a period, the probe will be triggered and Kubelet
 
 Note: because Citadel health checking currently only monitors the health status of CSR service API,
 this feature is not needed if the production setup is not using the
-[Istio Mesh Expansion](/docs/setup/kubernetes/mesh-expansion/) (which requires the CSR service API).
+[SDS](/docs/tasks/security/auth-sds/) or [Mesh Expansion](/docs/setup/kubernetes/mesh-expansion/).
```
Comment (rcaballeromx) [inline content/docs/tasks/security/health-check/index.md]:
```suggestion
[SDS](/docs/tasks/security/auth-sds/) or [Mesh Expansion](/docs/setup/kubernetes/additional-setup/mesh-expansion/).
```

File: content/docs/tasks/security/health-check/index.md [hunk 6]
```diff
@@ -22,74 +18,51 @@ If the file is not updated for a period, the probe will be triggered and Kubelet
...
 
 ## Before you begin
 
 * Set up Istio by following the instructions in the
-  [quick start](/docs/setup/kubernetes/quick-start/) with global mutual TLS enabled:
+  [Kubernetes quick start](/docs/setup/kubernetes/quick-start/) with global mutual TLS enabled:
```
Comment (rcaballeromx) [inline content/docs/tasks/security/health-check/index.md]:
```suggestion
  [Kubernetes installation guide](/docs/setup/kubernetes/install/kubernetes/). Remember to enable global mutual TLS with:
```

File: content/docs/tasks/security/health-check/index.md [hunk 7]
```diff
@@ -22,74 +18,51 @@ If the file is not updated for a period, the probe will be triggered and Kubelet
...
 
     {{< text bash >}}
     $ kubectl apply -f install/kubernetes/istio-demo-auth.yaml
     {{< /text >}}
 
     _**OR**_
 
-    Using [Helm](/docs/setup/kubernetes/helm-install/) with `global.mtls.enabled` to `true`.
+    Using [Helm](/docs/setup/kubernetes/helm-install/) with `global.mtls.enabled` set to `true`.
```
Comment (rcaballeromx) [inline content/docs/tasks/security/health-check/index.md]:
```suggestion
 * Use [Helm](/docs/setup/kubernetes/install/helm/) to setup Istio and set the `global.mtls.enabled` flag to `true`.
```

File: content/docs/tasks/security/health-check/index.md [hunk 8]
```diff
@@ -22,74 +18,51 @@ If the file is not updated for a period, the probe will be triggered and Kubelet
 
 Note: because Citadel health checking currently only monitors the health status of CSR service API,
 this feature is not needed if the production setup is not using the
-[Istio Mesh Expansion](/docs/setup/kubernetes/mesh-expansion/) (which requires the CSR service API).
+[SDS](/docs/tasks/security/auth-sds/) or [Mesh Expansion](/docs/setup/kubernetes/mesh-expansion/).
 
 ## Before you begin
 
 * Set up Istio by following the instructions in the
```
Comment (rcaballeromx) [inline content/docs/tasks/security/health-check/index.md]: Please introduce the list properly:
```suggestion
To complete this task, you can install Istio using one of the following paths:

* To setup Istio without using Helm, follow the instructions in the
```

File: content/docs/tasks/security/health-check/index.md [hunk 9]
```diff
@@ -22,74 +18,51 @@ If the file is not updated for a period, the probe will be triggered and Kubelet
...
-  [quick start](/docs/setup/kubernetes/quick-start/) with global mutual TLS enabled:
+  [Kubernetes quick start](/docs/setup/kubernetes/quick-start/) with global mutual TLS enabled:
 
     {{< text bash >}}
     $ kubectl apply -f install/kubernetes/istio-demo-auth.yaml
     {{< /text >}}
 
     _**OR**_
```
Comment (rcaballeromx) [inline content/docs/tasks/security/health-check/index.md]:
```suggestion
```

File: content/docs/tasks/security/health-check/index.md [hunk 10]
```diff
@@ -22,74 +18,51 @@ If the file is not updated for a period, the probe will be triggered and Kubelet
...
 
-    Using [Helm](/docs/setup/kubernetes/helm-install/) with `global.mtls.enabled` to `true`.
+    Using [Helm](/docs/setup/kubernetes/helm-install/) with `global.mtls.enabled` set to `true`.
 
 {{< tip >}}
-Starting with Istio 0.7, you can use [authentication policy](/docs/concepts/security/#authentication-policies) to configure mutual TLS for all/selected services in a namespace (repeated for all namespaces to get global setting). See [authentication policy task](/docs/tasks/security/authn-policy/)
+You can use [authentication policy](/docs/concepts/security/#authentication-policies) to configure mutual TLS for
```
Comment (rcaballeromx) [inline content/docs/tasks/security/health-check/index.md]:
```suggestion
Use an [authentication policy](/docs/concepts/security/#authentication-policies) to configure mutual TLS for
```

File: content/docs/tasks/security/health-check/index.md [hunk 11]
```diff
@@ -22,74 +18,51 @@ If the file is not updated for a period, the probe will be triggered and Kubelet
...
+all/selected services in a namespace (repeated for all namespaces to get global setting).
```
Comment (rcaballeromx) [inline content/docs/tasks/security/health-check/index.md]:
```suggestion
all or only selected services in a namespace. You must repeat the policy for all namespaces to configure the setting globally.
```

File: content/docs/tasks/security/health-check/index.md [hunk 12]
```diff
@@ -22,74 +18,51 @@ If the file is not updated for a period, the probe will be triggered and Kubelet
...
+See the [authentication policy task](/docs/tasks/security/authn-policy/) for details.
 {{< /tip >}}
 
 ## Deploying Citadel with health checking
 
-Deploy Citadel with health checking enabled.
+Redeploy Citadel with health checking enabled.
```
Comment (rcaballeromx) [inline content/docs/tasks/security/health-check/index.md]:
```suggestion
To enable health checking, redeploy Citadel using the configuration in `istio-citadel-with-health-check.yaml`:
```

File: content/docs/tasks/security/health-check/index.md [hunk 13]
```diff
@@ -22,74 +18,51 @@ If the file is not updated for a period, the probe will be triggered and Kubelet
...
-2018-02-27T04:30:11.081791Z     info    CSR successfully signed.
-...
-2018-02-27T04:30:25.485315Z     info    CSR successfully signed.
-...
+... CSR signing service is healthy (logged every 100 times).
 {{< /text >}}
 
 The log above indicates the periodic health checking is working.
-Observe that the health checking interval is about 15 seconds, which is the default health checking interval.
+The health checking interval is (by default) 15 seconds. It is logged every 100 times.
```
Comment (rcaballeromx) [inline content/docs/tasks/security/health-check/index.md]: The phrase "is logged every 100 times" is ambiguous. Do you mean: 

1. The health check is logged once every 100 checks, or
1. The health check is logged only on for the 100th, 200th, etc... check?

Assuming 1:

```suggestion
The default health checking interval is 15 seconds and is logged once every 100 checks.
```

File: content/docs/tasks/security/health-check/index.md [hunk 14]
```diff
@@ -113,13 +86,13 @@ livenessProbe:
 ...
 {{< /text >}}
 
-The `liveness-probe-path` and `probe-path` are the path to the health status file, configured at Citadel and the
-prober;
-the `liveness-probe-interval` is the interval to update the health status file, if Citadel is healthy;
-the `probe-check-interval` is the interval for Citadel health checking.
+The `liveness-probe-path` and `probe-path` are the paths to the health status file, configured at Citadel and the
```
Comment (rcaballeromx) [inline content/docs/tasks/security/health-check/index.md]:
```suggestion
The paths to the health satus files are `liveness-probe-path` and `probe-path`. You can configure the paths in Citadel and in the
```

File: content/docs/tasks/security/health-check/index.md [hunk 15]
```diff
@@ -113,13 +86,13 @@ livenessProbe:
...
+prober.
+The `liveness-probe-interval` is the interval to update the health status file (if Citadel is healthy).
```
Comment (rcaballeromx) [inline content/docs/tasks/security/health-check/index.md]:
```suggestion
If Citadel is healthy, the value of the `liveness-probe-interval` entry determines the interval used to update the health status file.
```

File: content/docs/tasks/security/health-check/index.md [hunk 16]
```diff
@@ -113,13 +86,13 @@ livenessProbe:
...
+The `probe-check-interval` is the interval for Citadel health checking controller to call the Citadel CSR service.
```
Comment (rcaballeromx) [inline content/docs/tasks/security/health-check/index.md]:
```suggestion
The Citadel health checking controller uses the value of the `probe-check-interval` entry to determine the interval used to call the Citadel CSR service.
```

File: content/docs/tasks/security/health-check/index.md [hunk 17]
```diff
@@ -113,13 +86,13 @@ livenessProbe:
...
 The `interval` is the maximum time elapsed since the last update of the health status file, for the prober to consider
 Citadel as healthy.
-`initialDelaySeconds` and `periodSeconds` are the initial delay and the probe running period.
+`initialDelaySeconds` and `periodSeconds` are the initial delay and the interval between activations of the probe.
```
Comment (rcaballeromx) [inline content/docs/tasks/security/health-check/index.md]:
```suggestion
The values in the `initialDelaySeconds` and `periodSeconds`entries determine the initial delay and the interval between each activation of the probe.
```

**Comment by myidpt** [comment]
Thanks @rcaballeromx, could you approve?

**Pull request merged**